src/content/docs/logs/forward-logs/vercel-integration.mdx

```diff
@@ -0,0 +1,148 @@
+    title="Step 1. Add the integration"
+    >
+    Open the [Vercel's New Relic integration page](https://vercel.com/integrations/newrelic) and click **Add integration**.
+
+    <img
+    width="50%;"
+    title="Vercel's New Relic integration page"
+    alt="Vercel's New Relic integration page"
+    src={logsVercelIntegration}
+  />
+
+</Collapser>
+<Collapser
+    id="step2"
+    title="Step 2. Connect the Vercel account"
+    >
+    Select the Vercel account you'd like to connect and click **Continue**.
+
+    <img
+    width="50%;"
+    title="Select the Vercel account"
+    alt="Select the Vercel account"
+    src={logsVercelAccount}
+  />
+
+</Collapser>
+<Collapser
+    id="step3"
+    title="Step 3. Select projects"
+    >
+    Select which projects you want to send logs from and click **Continue**.
+
+    <img
+    width="50%;"
+    title="Select the Vercel projects"
+    alt="Select the Vercel projects"
+    src={logsVercelProjects}
+  />
+
+</Collapser>
+<Collapser
+    id="step4"
+    title="Step 4. Allow permissions"
+    >
+    Allow permissions to read and write for log drains and click **Add integration**.
+
+    <img
+    width="50%;"
+    title="Allow permissions"
+    alt="Allow permissions"
+    src={logsVercelAllowPermissions}
+  />
+
+</Collapser>
+<Collapser
+    id="step5"
+    title="Step 5. Set your license key"
+    >
+    Go to the [API Keys UI](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher) on your New Relic dashboard to create or retrieve your license key.
+
+    Copy the key, paste it in the **Your New Relic license key** field, and click **Add Integration**.
+
+    <img
+    width="50%;"
+    title="Set License Key"
+    alt="Set License Key"
+    src={logsVercelSetLicenseKey}
+  />
+</Collapser>
+</CollapserGroup>
+
+
+## Install the Vercel quickstart dashboard [#vercel-quickstart]
+
+After you've sent Vercel data to New Relic, the Vercel quickstart is the fastest way to jump straight into monitoring your end-to-end development logs.
+
+Install the quickstart to get a pre-built dashboard and alert to monitor the performance of your functions. You'll also get a pre-built alert that triggers when the error rate (4xx/5xx errors) exceeds 1%. Follow these instructions:
+
+1. Go to the [Vercel quickstart](https://newrelic.com/instant-observability/vercel) in New Relic Instant Observability and click **Install now**.
+2. Select an account and click **Begin installation**.
+3. If you've already completed the [Vercel configuration steps](#configuration), select **Done** to move onto the next step.
+4. The quickstart deploys the resources to your account. Click **See your data** to get to the dashboard.
+
+The Vercel quickstart dashboard visualizes all your key data into three pages. Click the **Overview**, **Build Logs**, and **Serverless Function** tabs to get an overview, analyze build logs, and dive deep into your Serverless Function performance.
+
+**Add here the dashboard screenshot**
```

Comment:
???

Reply:
Yes, it is. I'm waiting for an environment and take it.